Gemfile.rails_version

```diff
@@ -2,13 +2,15 @@
 # separate fragment so that it can be sourced from the test
 # application's Gemfile in addition to the main development Gemfile.
 
-env_rails_version = ENV["RAILS_VERSION"] || '4.2.10'
+env_rails_version = ENV['RAILS_VERSION'] || '5.1.7'
 
 gem "rails", "~> #{env_rails_version}"
 
 case env_rails_version
 when "master"
   {github: "rails/rails"}
+when /^5.1/
+  gem 'simple_form'
```

Comment:
Why do we still need this? Can we remove it?

Reply:
I think so- there are a few initializers that configure simple form but I think we can remove those as well because I deleted the haml files that have simple_forms